pkg/install/install.go

```diff
@@ -603,3 +673,53 @@ func (o *Install) installOperator(ctx context.Context, channel, operatorName str
 func (o *Install) restartEverestOperatorPod(ctx context.Context) error {
 	return o.kubeClient.RestartEverest(ctx, "everest-operator", o.config.Namespace)
 }
+
+func (o *Install) serviceAccountRolePolicyRules() []rbacv1.PolicyRule {
+	return []rbacv1.PolicyRule{
+		{
+			APIGroups: []string{"everest.percona.com"},
+			Resources: []string{"databaseclusters", "databaseclusterrestores"},
```

Comment:
Why do we have `databaseclusterrestores` here if it's also specified below?
```suggestion
			Resources: []string{"databaseclusters"},
```